common/persistence/persistence-tests/executionManagerTestForEventsV2.go

```diff
@@ -124,15 +124,15 @@ func (s *ExecutionManagerSuiteForEventsV2) TestWorkflowCreation() {
 				RunID:                      workflowExecution.GetRunId(),
 				TaskQueue:                  "taskQueue",
 				WorkflowTypeName:           "wType",
-				WorkflowRunTimeout:         20,
-				DefaultWorkflowTaskTimeout: 13,
+				WorkflowRunTimeout:         timestamp.DurationFromSeconds(20),
```

Comment:
Imho, I prefer having `DurationFromUnit()` to avoid mistakes in math and make refactoring a bit simpler. Can always drop down to `DurationPtr() with more complex times. Lets chat more.